### DISCUSSION TITLE
webots.cloud leaderboard page doesnt load data

### DISCUSSION THREAD

**Discussion by JustLuckyRobotics** [cyberbotics/wrestling#64] [General]
All day today i have been unable to view the leaderboard.

I have noticed that when we are developing on evaluation server, sometimes if i refresh too much a DDOS style protection kicks in for a about 20-30 minutes but usually comes back. however today i have tried on multiple pc's, on my phone, on multiple WIFI connections, while using vpns, all with no success. 

My team mate is also suffering from the same issues except on his ubuntu install. windows and android appear to be bricked or the DDOS protection has gotten more aggressive.

**Comment by omichel** [comment]
Thanks for reporting this. It should be fixed by now.
Regarding the DDOS style problems, they are related to the limited number of anonymous requests sent to GitHub. We are aware of the problem and will try to fix it soon.

**Reply by omichel** [reply]
I believe we have now fixed the DDOS style problems. Can you confirm?

**Reply by JustLuckyRobotics** [reply]
can confirm, not noticed any such issues for a while.
